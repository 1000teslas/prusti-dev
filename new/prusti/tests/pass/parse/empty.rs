// compile-flags: -Zprint-desugared-specs -Zskip-verify

fn main() {}
