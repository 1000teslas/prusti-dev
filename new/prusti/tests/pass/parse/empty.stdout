#[prelude_import]
use std::prelude::v1::*;
#[macro_use]
extern crate std;
// compile-flags: -Zprint-desugared-specs

fn main() { }
HIR Item: 
  Not a function
HIR Item: std
  Not a function
HIR Item: main
  Return type is unit: true
